src/common/src/array/data_chunk.rs

```diff
@@ -62,7 +62,7 @@ impl DataChunkBuilder {
 }
 
 /// `DataChunk` is a collection of arrays with visibility mask.
-#[derive(Clone, Default)]
+#[derive(Clone, Default, PartialEq)]
```

Comment:
Should we consider comparing two data chunks regardless of the visibility? That is, even though the visibilities differ, they are the same since the data after compaction is the same.